build/makelib/golang.mk

```diff
@@ -177,6 +183,12 @@ $(GO_TOOLS_DIR)/glide:
 	@mv $(GO_TOOLS_DIR)/$(GOHOSTOS)-$(GOHOSTARCH)/glide $(GO_TOOLS_DIR)/glide
 	@rm -r $(GO_TOOLS_DIR)/$(GOHOSTOS)-$(GOHOSTARCH)
 
+.PHONY: $(GO_TOOLS_DIR)/golint
```

Comment:
if you want to support cross platforms then maybe put it in `$(GO_TOOLS_DIR)/$(GOOS)_$(GOARCH)`. With phony this is going install lint every time which is not right.